lib/widgets/receivable_btc_box.dart

```diff
@@ -69,33 +69,33 @@ class FeeMessage extends StatelessWidget {
     final themeData = Theme.of(context);
     return BlocBuilder<LSPBloc, LspState?>(builder: (context, lspState) {
       final lspInfo = lspState?.lspInfo;
-      return lspInfo == null
-          ? const SizedBox()
-          : WarningBox(
+      return lspInfo != null && lspInfo.openingFeeParamsList.values.isNotEmpty
```

Comment:
Will openingFeeParamsList.values be empty if there are not fees?

Reply:
Regarding displaying the fees do we wish to display set fees when `liquidity > maxAllowedToReceive` ?

Reply:
Yes, that's true because the maximum amount we can receive is exempt from fees in that case.